src/go/rpk/pkg/cli/cmd/cluster/txn/abort.go

```diff
@@ -0,0 +1,106 @@
+// Copyright 2022 Redpanda Data, Inc.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.md
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0
+
+package txn
+
+import (
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/config"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/kafka"
+	"github.com/redpanda-data/redpanda/src/go/rpk/pkg/out"
+	"github.com/spf13/afero"
+	"github.com/spf13/cobra"
+	"github.com/twmb/franz-go/pkg/kadm"
+)
+
+func newAbortCommand(fs afero.Fs) *cobra.Command {
+	var (
+		topic       string
+		partition   int32
+		startOffset int64
+	)
+	cmd := &cobra.Command{
+		Use:   "abort",
+		Short: "Abort an open transaction for a given partition",
+		Long: `Abort an open transaction for a given partition.
+
+This command aborts a single partition in a transaction, allowing the
+last-stable-offset to advance for the partition. To be extra cautious, this
+command only aborts one partition at a time. You can run this repeatedly to
+abort all partitions that may be hanging in a single transaction.
+
+You can find potentially hanging transactions with the find-hanging command.
+
+`,
+
+		Run: func(cmd *cobra.Command, txnIDs []string) {
+			if topic == "" {
+				out.Die("missing required --topic")
+			}
+			if partition < 0 {
+				out.Die("missing required --partition")
+			}
+			if startOffset < 0 {
+				out.Die("missing required --start-offset")
+			}
+
+			p := config.ParamsFromCommand(cmd)
+			cfg, err := p.Load(fs)
+			out.MaybeDie(err, "unable to load config: %v", err)
+
+			adm, err := kafka.NewAdmin(fs, p, cfg)
+			out.MaybeDie(err, "unable to initialize kafka client: %v", err)
+			defer adm.Close()
+
+			var s kadm.TopicsSet
+			s.Add(topic, partition)
+			producers, err := adm.DescribeProducers(cmd.Context(), s)
+			out.HandleShardError("DescribeProducers", err)
+
+			var hung kadm.DescribedProducer
+			var found bool
```

Comment:
```suggestion
			var (
				hung  kadm.DescribedProducer
				found bool
			)
```

Wdyt, should we add a linter for this?  https://golangci-lint.run/usage/linters/#grouper (I do think that 2 vars are fine but I've seen the grouping for 2 in other parts of the codebase)

Reply:
I think that with two, it's somewhat of a matter of preference -- I haven't really cared about two in the past. I'd be wary of how overkill that linter could be, as well, but it may be worth evaluating.